### ISSUE TITLE
[Failing Test] post-release-push-image-deb-builder (post-release-push-image-deb-builder)

### ISSUE THREAD

**Issue by droslean** [kubernetes/kubernetes#87141]
**Which jobs are failing**:
`post-release-push-image-deb-builder (post-release-push-image-deb-builder)`

**Testgrid link**:
https://testgrid.k8s.io/sig-release-master-informing#post-release-push-image-deb-builder

**Reason for failure**:
```console
 Activating service account...
Activated service account credentials for: [deployer@k8s-prow.iam.gserviceaccount.com]
Running...
2020/01/12 17:46:00 Build directory: build/debs
2020/01/12 17:46:00 Config directory (/home/prow/go/src/github.com/kubernetes/release/build/debs) does not exist
```

**Anything else we need to know**:
/cc @kubernetes/ci-signal 
/cc @kubernetes/release-managers 
/sig release
/milestone v1.18

This is a postsubmit job for `kubernetes/release` repository.

**Comment by justaugustus** [comment]
(deb counterpart to https://github.com/kubernetes/kubernetes/issues/86965)

https://github.com/kubernetes/release/pull/1011 removed the `build/` dir to consolidate the `kubepkg` assets in one dir.
This should be fixed once kubernetes/test-infra#15871 merges.

/assign
/priority important-soon

**Comment by justaugustus** [comment]
Closed via https://github.com/kubernetes/test-infra/pull/15871.
/close
